.vscode/settings.json

```diff
@@ -9,4 +9,40 @@
 	],
 	"cmake.configureOnEdit": false,
 	"cmake.configureOnOpen": false,
+	"files.associations": {
```

Comment:
This has been a problem for a while - vscode likes generating a random list here : (
Can you add 
```json
    "C_Cpp.autoAddFileAssociations": false
```
instead of the `"files.associations"` key?